app/helpers/application_helper/toolbar/openstack_vm_cloud_center.rb

```diff
@@ -58,13 +58,13 @@ class ApplicationHelper::Toolbar::OpenstackVmCloudCenter < ApplicationHelper::To
           :klass => ApplicationHelper::Button::InstanceDetach),
         button(
           :instance_associate_floating_ip,
-          'fa fa-map-marker fa-lg',
+          'ff ff-floating-ip fa-lg',
           t = N_('Associate a Floating IP with this Instance'),
           t,
           :klass => ApplicationHelper::Button::InstanceAssociateFloatingIp),
         button(
           :instance_disassociate_floating_ip,
-          'fa fa-map-marker fa-lg',
+          'ff ff-floating-ip',
```

Comment:
Either this one is missing `fa-lg` now or the previous button should not have it either ;)